dotcom-rendering/src/model/enhance-numbered-lists.test.ts

```diff
@@ -4,14 +4,16 @@ import { images } from '../../fixtures/generated/images';
 import { blockMetaData } from '../../fixtures/manual/block-meta-data';
 import { enhanceNumberedLists } from './enhance-numbered-lists';
 
+const [image] = images as [ImageBlockElement];
+
```

Comment:
This is a test, so failures will be caught, if the type assertion is not correct.